lib/bolt_server/transport_app.rb

```diff
@@ -78,7 +84,13 @@ def initialize(config)
 
       body = JSON.parse(request.body.read)
       schema_error = JSON::Validator.fully_validate(@schemas["winrm-run_task"], body)
-      return [400, schema_error.join] if schema_error.any?
+      if schema_error.any?
+        msg = "There was an error validating the request body. Please see for expected schema"
```

Comment:
Was this supposed to be different from line 58?

Reply:
Good catch - fixed to match!